scripts/noshake.json

```diff
@@ -404,6 +404,7 @@
   ["Mathlib.AlgebraicTopology.AlternatingFaceMapComplex"],
   "Mathlib.Algebra.Star.Module": ["Mathlib.Algebra.Module.LinearMap.Star"],
   "Mathlib.Algebra.Ring.CentroidHom": ["Mathlib.Algebra.Algebra.Defs"],
+  "Mathlib.Algebra.Order.Module.Defs": ["Mathlib.Algebra.Order.Module.Synonym"],
```

Comment:
Why is this necessary?

Reply:
It was necessary at some point for some reason, but it looks it's not anymore